packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -60,8 +60,21 @@ if (__DEV__) {
 // this module is initially evaluated.
 // We want to be using a consistent implementation.
 const localDate = Date;
-const localSetTimeout = setTimeout;
-const localClearTimeout = clearTimeout;
+const localSetTimeout =
+  typeof setTimeout === 'function'
+    ? setTimeout
+    : warning(
```

Comment:
This looks a bit misleading, as if we’re using its return value.

Let’s just keep undefined there explicitly and then warn later.